refgenie/refgenie.py

```diff
@@ -29,7 +29,7 @@
 INSERT_CMD = "add"
 
 
-BUILD_SPECIFIC_ARGS = ('fasta', 'gtf', 'context')
+BUILD_SPECIFIC_ARGS = ('fasta', 'gtf', 'context', 'annogene')
```

Comment:
what is the annogene exactly? is it not the same as the gtf file?

Reply:
refGene.txt.gz --> refGene.txt

Reply:
https://github.com/databio/pepatac/blob/dev/docs/faq.md#how-do-i-calculate-tss-enrichments